File: Backend/src/main/java/com/example/Billing/System/io/OrderRequest.java
package com.example.Billing.System.io;

//import java.time.LocalDateTime;
import java.util.List;

//import com.example.Billing.System.io.OrderResponse.OrderItemResponse;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class OrderRequest {
private String customerName;
private String phoneNumber;
private List<OrderItemRequest> cartItems;
private Double subtotal;
private Double tax;
private Double grandTotal;
private PaymentMethod paymentMethod;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public static class OrderItemRequest{
	private String itemId;
	private String name;
	private Double price;
	private Integer quantity;
}

	
}

File: Frontend/src/app/io/OrderRequest.ts
export interface OrderItemRequest {
  itemId: string;
  name: string;
  price: number;
  quantity: number;
}
export interface OrderRequest {
  customerName: string;
  phoneNumber: string;
  cartItems: OrderItemRequest[];
  subtotal: number;
  tax: number;
  grandTotal: number;
  paymentMethod: string; // Use 'CASH' or 'UPI'
}